### ISSUE TITLE
Separate High Confidence from Low Confidence genes

### ISSUE THREAD

**Issue by rsuchecki** [plantinformatics/pretzel-input-generator#12]
Some assemblies come with two subsets of gene predictions, High Confidence (HC) and Low Confidence (LC). LC predictions are often dubious so best not to have them mixed together with HC. Should probably define genome with HC as a separate entity from genome + LC, for example:

```
    localAssembly = [
      [ 
        species : "Triticum_dicoccoides",
        version : "WEWseq_PGSB_20160501_HC",
...
```

```
    localAssembly = [
      [ 
        species : "Triticum_dicoccoides",
        version : "WEWseq_PGSB_20160501_LC",
...
```

**Comment by rsuchecki** [comment]
This is not quite right, this should really be one genome with 2 separate annotations. This will require updates to accommodate multiple annotations.

**Comment by rsuchecki** [comment]
available in [v0.5](https://github.com/plantinformatics/pretzel-input-generator/releases/tag/v0.5)
